pkg/resources/aws/eni.go

```diff
@@ -66,24 +66,28 @@ func DumpENI(op *resources.DumpOperation, r *resources.Resource) error {
 func DescribeENIs(cloud fi.Cloud, clusterName string) (map[string]*ec2.NetworkInterface, error) {
 	c := cloud.(awsup.AWSCloud)
 
+	statusFilter := &ec2.Filter{
+		Name: aws.String("status"),
+		Values: []*string{
+			aws.String(ec2.NetworkInterfaceStatusDetaching),
```

Comment:
Do we care about the detaching interfaces in our case?
I think the List function is called again and again, so those detaching ENIs will become available at some point.

Reply:
No, ENIs that aren't returned won't get a resource tracker assigned. I suppose the assumption is that ENIs that are attached will be implicitly deleted once the thing they're attached to is deleted.